com.nickgirga.webready.yml

```diff
@@ -0,0 +1,49 @@
+id: com.nickgirga.webready
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: webready
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # access to home files; for locating WADs and custom installed source ports
+  - --filesystem=home
+
+modules:
+  - name: webready
+    buildsystem: simple
+    build-commands:
+      - install -D webready /app/bin/webready
+      - install -D webready.glade /app/share/webready/webready.glade
+      - install -D res/icons/scalable.svg /app/share/icons/hicolor/scalable/apps/com.nickgirga.webready.svg
+      - install -D res/icons/128x.png /app/share/icons/hicolor/128x128/apps/com.nickgirga.webready.png
+      - install -D com.nickgirga.webready.desktop /app/share/applications/com.nickgirga.webready.desktop
+      - install -Dm644 com.nickgirga.webready.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: git
+        url: https://gitlab.com/nickgirga/webready.git
+        tag: 1.0.1
+
+  - name: ffmpeg_progress_yield
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/ec/42/4d4a0471c56e7c493bb919b3518a7390f2f6b5110fd00bde909f97883807/ffmpeg-progress-yield-0.7.8.tar.gz
+        sha256: 9ae6ae5f832ae7c7b0f6518f1341fe6eee1ba8fc9d3400cba1cba38f2eaa461e
+
+  - name: libhandy
+    buildsystem: meson
+    config-opts:
+      - -Dglade_catalog=disabled
+      - -Dtests=false
+      - -Dexamples=false
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libhandy
+        tag: 1.0.3
```

Comment:
Do you need 1.0.3? Runtime has 1.8.3

Reply:
Oh, great. I didn't know that. I will remove libhandy as a module.